ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -424,11 +424,12 @@ def _generate_step_input(
             if n_agents == 0:
                 continue
             for i in range(n_agents):
-                # TODO: extend to AgentBuffers
+                _act = []
```

Comment:
```suggestion
                # TODO add separate fields for continuous and discrete actions in AgentActionProto
                _act = []
```
(I hope I got the spaces lined up on that)